webpack/assets/javascripts/react_app/components/PF4/InlineEdit/InlineEdit.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import {
+  Spinner,
+  Text,
+  TextVariants,
+  Button,
+  Split,
+  SplitItem,
+} from '@patternfly/react-core';
+import { TimesIcon, CheckIcon, PencilAltIcon } from '@patternfly/react-icons';
+import PropTypes from 'prop-types';
+import InlineTextInput from './InlineTextInput';
+import { translate as __ } from '../../../common/I18n';
+import './inlineEdit.scss';
+
+const InlineEdit = ({ onEdit, value, textArea, attribute }) => {
+  // Tracks input box state
+  const [inputValue, setInputValue] = useState(value);
+  const [editing, setEditing] = useState(false);
+  const [submitting, setSubmitting] = useState(false);
+
+  const onSubmit = async () => {
+    setSubmitting(true);
+    await onEdit(inputValue, attribute);
+    setSubmitting(false);
+    setEditing(false);
+  };
+
+  const onClear = () => {
+    setInputValue(value);
```

Comment:
This was happening because the hooks were not inside a react component, just the storybook, so their updates were not triggering a re-render. I modified to use `React.createElement` which seems to work quite well. Let me know what you think!

Reply:
(this is probably where the storybook eslint hooks rule comes from)

Reply:
I did the same in `ForemanModal` stories, nice :)